airgun/navigation.py

```diff
@@ -45,6 +45,10 @@ def am_i_here(self, *args, **kwargs):
 
     def go(self, _tries=0, *args, **kwargs):
         super(BaseNavigator, self).go(_tries=_tries, *args, **kwargs)
+        try:
+            self.prerequisite_view = self.parent
```

Comment:
Is there any functionality tied to `self.prerequisite_view`? Or it's only reason is to have a nice shortcut without need of using getattr?

Reply:
Asking as i personally like `self.parent` more, as it's already there with no properties duplication and it's basically shorter. If there's no functional meaning we could keep using it, just try to update it here so it always available.

Reply:
We use prerequisite_view to make it visually clear that it is the view from the previous step, which you usually use to navigate to the location the class represents inside the step()

Reply:
Not sure about this change then. For me `self.parent` looks visually clearer than `self.prerequisite_view` and doesn't introduce another attribute pointing to the same object. @oshtaier your thoughts? Should we follow the same approach?